Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1

```diff
@@ -32,6 +32,18 @@ function Get-xSharePointContentService() {
     return [Microsoft.SharePoint.Administration.SPWebService]::ContentService
 }
 
+
+function Get-xSharePointUserProfileSubTypeManager {
+    [CmdletBinding()]
+    param
+    (
+        $context
```

Comment:
Function parameter names should be Pascal case per [style guidance](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#parameter-names-should-use-pascalcase-)